package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "npm ci && npm run bundle && npm run format",
-    "bundle": "esbuild ./src/main.js --bundle --platform=node --outfile=dist/index.js --tsconfig-raw='{}'",
+    "bundle": "esbuild ./src/main.js --bundle --platform=node --outfile=dist/index.js",
```

Comment:
I removed this because it actually caused errors when I was trying to build locally without it.

Reply:
I'm curious why you were running into errors. Based on [my own research](https://esbuild.github.io/api/#tsconfig-raw) and what Steve said when he added it, that argument shouldn't affect anything since we're not using typescript.

Really it shouldn't make a difference having that argument or not having it, so I'm surprised you ran into problems and those problems were fixed by removing it.

Reply:
![image](https://github.com/im-open/process-cypress-test-results/assets/2112531/5bcfdabd-0506-4e9c-b016-0fc55c9657e8)

I can put it back in though with double quotes.